docs/pages/typography.js

```diff
@@ -0,0 +1,1008 @@
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Regular</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Box color="darkGray" rounding={2} padding={2} display="inlineBlock">
+                    <Text color="inverse" size="100">
+                      This is text in a tooltip
+                    </Text>
+                  </Box>
+                </Table.Cell>
+              </Table.Row>
+              <Table.Row>
+                <Table.Cell>
+                  <Text size="200">Emphasis for in-line text</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Bold</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Semibold</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Medium</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Box color="darkGray" rounding={2} padding={2} display="inlineBlock">
+                    <Text color="inverse" size="100" overflow="noWrap" inline>
+                      Figma is{' '}
+                      <Text weight="bold" color="inverse" size="100" overflow="noWrap" inline>
+                        not a design system
+                      </Text>
+                    </Text>
+                  </Box>
+                </Table.Cell>
+              </Table.Row>
+              <Table.Row>
+                <Table.Cell>
+                  <Text size="200">Stand-alone links</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Bold</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Semibold</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Medium</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Flex gap={8}>
+                    <Text weight="bold">
+                      <Link href="https://pinterest.com">Pinterest</Link>
+                    </Text>
+                    <Text weight="bold">
+                      <Link href="https://gestalt.pinterest.systems">Gestalt</Link>
+                    </Text>
+                  </Flex>
+                </Table.Cell>
+              </Table.Row>
+              <Table.Row>
+                <Table.Cell>
+                  <Text size="200">Headlines</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Bold</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Semibold</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">Medium</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Heading>Page title</Heading>
```

Comment:
I'd use Text here with the biggest font size, to pass the a11y test

Reply:
See comment below

Reply:
Done!